stdlib/2and3/poplib.pyi

```diff
@@ -0,0 +1,68 @@
+# Stubs for poplib (Python 2 and 3)
+
+from mypy_extensions import NoReturn
+import socket
+import ssl
+import sys
+from typing import Any, BinaryIO, Dict, List, Optional, Pattern, Text, Tuple
+
+class error_proto(Exception): pass
+
+POP3_PORT: int
+POP3_SSL_PORT: int
+CR: bytes
+LF: bytes
+CRLF: bytes
+
+
+class POP3:
+    if sys.version_info >= (3, 0):
+        encoding: Text
+
+    host: Text
+    port: int
+    sock: socket.socket
+    file: BinaryIO
+    welcome: bytes
+
+    def __init__(self, host: Text, port: int = ..., timeout: float = ...) -> None: ...
+    def getwelcome(self) -> bytes: ...
+    def set_debuglevel(self, level: int) -> None: ...
+    def user(self, user: Text) -> bytes: ...
+    def pass_(self, pswd: Text) -> bytes: ...
+    def stat(self) -> Tuple[int, int]: ...
+    def list(self, which: Optional[Text] = ...) -> bytes: ...
+    def retr(self, which: Text) -> bytes: ...
```

Comment:
As for the argument type, it really accepts anything because the only thing it does with the argument is "%s". I think I put Text because that's the only type that's actually used in the tests. I think I'll just make it Any instead to be safe.